### PR TITLE
[1LP][RFR] Blocking test_run_host_analysis with BZ1650179

### PR DESCRIPTION
**Pull request by jan-zmeskal**
Blocker

### CODE REVIEW

File: cfme/tests/infrastructure/test_host_analysis.py [hunk 1]
```diff
@@ -63,6 +63,8 @@ def host_with_credentials(provider, host_name):
 @pytest.mark.uncollectif(
     lambda provider, appliance:
     appliance.version == miq_version.UPSTREAM and provider.one_of(RHEVMProvider))
+@pytest.mark.meta(blockers=[BZ(1650179, forced_streams=['5.10'],
```
Comment (mshriver) [inline cfme/tests/infrastructure/test_host_analysis.py]: Gotta import BZ, wasn't already imported in this module. Why travis is failing. :)

Comment (jan-zmeskal) [reply cfme/tests/infrastructure/test_host_analysis.py]: Nice catch, thank you!

**Comment by jawatts** [comment]
@jan-zmeskal PR changes look good but have you seen the failures for 59z?

**Comment by jan-zmeskal** [comment]
@jawatts I don't know what do you mean. From the looks of it it seems that the test passes for rhv41.

**Pull request merged**